src/frontend/packages/lib_video/src/components/common/Player/createVideojsPlayer.ts

```diff
@@ -51,8 +51,7 @@ export const createVideojsPlayer = (
   // add the video-js class name to the video attribute.
   videoNode.classList.add('video-js', 'vjs-big-play-centered');
 
-  const isLive =
-    video.live_state !== null && video.live_state !== liveState.ENDED;
+  const isLive = video.is_live;
```

Comment:
IMO there is no need anymore to store it in a variable. Before yes, it avoids to compute the value again and again. This is not the case anymore.